modules/secrets-store-auto-rotation.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-secrets-store.adoc
+
+:_content-type: CONCEPT
+[id="secrets-store-auto-rotation_{context}"]
+= Automatic rotation
+
+The {secrets-store-driver} periodically rotates the content in the mounted volume with the content from the external secrets store. If a secret is updated in the external secrets store, the secret will be updated in the mounted volume. The {secrets-store-operator} polls for updates every 2 minutes.
+
+If you enabled synchronization of mounted content as Kubernetes secrets, the Kubernetes secrets are also rotated.
+
+Applications consuming the secret data must watch for updates to the secrets.
```

Comment:
Is it safe to assume that anyone reading this concept info will just know how to do that for their application? Would it be helpful to include a recommended method for an application to monitor for updates? Or maybe an example method here would be useful?

Reply:
@dobsonj Thoughts on this question? Do we need to get into any details on how to do this, or can users figure that out for their apps easy enough?

Reply:
IMO, that will be so application specific that it's not worth going into details here.

The [upstream doc on auto rotation](https://secrets-store-csi-driver.sigs.k8s.io/topics/secret-auto-rotation) mentions a few options, but it's really up to the application developer to decide on a method that will work.

Reply:
Okay cool, thank you!